src/openproduct/producttypen/viewsets/thema.py

```diff
@@ -9,10 +9,15 @@
 from openproduct.logging.api_tools import AuditTrailViewSetMixin
 from openproduct.producttypen.models import ProductType, Thema
 from openproduct.producttypen.serializers import ThemaSerializer
-from openproduct.utils.filters import FilterSet
+from openproduct.utils.filters import FilterSet, UUIDFInFilter
 
 
 class ThemaFilterSet(FilterSet):
+    producttypen__uuid__in = UUIDFInFilter(
+        field_name="producttypen__uuid",
+        lookup_expr="in",
+        help_text="Filter thema's op basis van een lijst met producttype uuid's.",
+    )
```

Comment:
:rotating_light: distinct=True is needed otherwise duplicate thema's can be returned. 

```py
            producttype = ProductTypeFactory.create()
            producttype_2 = ProductTypeFactory.create()

            thema = ThemaFactory.create()
            thema.producttypen.set([producttype, producttype_2])
            thema.save()

            response = self.client.get(self.path, {"producttypen__uuid__in": f"{producttype.uuid},{producttype_2.uuid}"}
            )

            self.assertEqual(response.status_code, status.HTTP_200_OK)
            self.assertEqual(response.data["count"], 1) 
```